DATS.json

```diff
@@ -116,7 +116,7 @@
 			"category": "files",
 			"values": [
 				{
-					"value": "317"
+					"value": "316"
```

Comment:
@mathdugre how come there is one less file than before in the DATS description but two additional files in the PR?

Reply:
@cmadjar Should the number of file include only annexed files? Or also non-annexed files?

Reply:
I would tend to include all the files. @emmetaobrien thoughts on this?